site/en/blog/css-in-js/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "CSS-in-JS support in DevTools"
+description: >
+  How we support CSS-in-JS in DevTools and how it is different from regular CSS.
+layout: "layouts/blog-post.njk"
+authors:
+  - orkon
+date: 2021-02-26
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fD43v6RiVCDlmo3WhVPe.jpg'
+alt: >
+  DevTools blog image
```

Comment:
Removed

Reply:
The lint complain about undefined alt, so I am adding an empty alt `alt: ''` for the moment.

Reply:
I noticed LHCI was complaining because we had a link around an image without alt text so I pushed a small change in https://github.com/GoogleChrome/developer.chrome.com/pull/587/commits/60bde584aaa6c9369fc3cda102c4f354fc64083e which will remove the image link from the accessibility tree if it doesn't have an `alt`. The card still has a clickable heading which should still work for screen reader access.